mitodl/mitxonline
# Updates to prefer long bio over short for existing design

# What are the relevant tickets?

Closes #1811 

# Description (What does it do?)

* Adds a migration to migrate instructor short bios into the long bio field (this will overwrite all long bios (but there aren't any right now))
* Updates the template to display the long bio rather than the short one
* Updates the `convert_faculty_to_pages` command to populate the short bio with the instructor name rather than the bio (to prevent confusion)

# How can this be tested?

1. Change one of the short bios so you can tell that the migration worked.
2. Run the migration and check to make sure instructor info shows up as expected (esp. with the page that was edited in step 1).
3. Add an old-style faculty member to a course.
4. Run `convert_faculty_to_pages` - the new page should have the bio in the long field.
